wkcuber/api/view.py

```diff
@@ -433,17 +436,39 @@ def _handle_compressed_write(
             tuple(aligned_offset) != absolute_offset
             or tuple(aligned_shape) != data.shape[-3:]
         ):
-            # the data is not aligned
-            # read the aligned bounding box
-
-            # We want to read the data at the absolute offset.
-            # The absolute offset might be outside of the current view.
-            # That is the case if the data is compressed but the view does not include the whole file on disk.
-            # In this case we avoid checking the bounds because the aligned_offset and aligned_shape are calculated internally.
-            warnings.warn(
-                "Warning: write() was called on a compressed mag without block alignment. Performance will be degraded as the data has to be padded first.",
-                RuntimeWarning,
+            mag_view_bbox_at_creation = self._mag_view_bbox
+
+            # Calculate in which dimensions the data is aligned and in which dimensions it matches the bbox of the mag.
+            aligned_top_left = aligned_offset == np.array(absolute_offset)
+            aligned_bottom_right = aligned_bottom_right == bottom_right
```

Comment:
```suggestion
            is_top_left_aligned = aligned_offset == np.array(absolute_offset)
            is_bottom_right_aligned = aligned_bottom_right == bottom_right
```

would be better imo (e.g., `aligned_offset` has currently the same naming scheme and isn't a boolean).